src/http_server.cc

```diff
@@ -3225,6 +3225,42 @@ HTTPAPIServer::HandleGenerate(
       req, RestrictedCategory::INFERENCE, restricted_apis_);
 
   AddContentTypeHeader(req, "application/json");
+#ifdef TRITON_ENABLE_METRICS
+  // logic to add kv_cache metrics to response header
+  // Get the metrics in Prometheus format
+  if (std::getenv("ORCA_HEADER_METRIC_TYPE") != nullptr) {
```

Comment:
Change the name of the env var to `ORCA_METRIC_FORMAT`, and pls document the usage of it, also link the ORCA proposal doc here for reference

Reply:
Usage documented in related [feature request issue](https://github.com/triton-inference-server/server/issues/7865) and in code comments along with link.